### PR TITLE
Remove flatbuf tar.gz from tensorflow

### PR DESCRIPTION
**Pull request by gichan-jang**
NNStreamer ppa already have the flatbuffers package, so don't download additional flatbuf tar.gz.

Newly packaged tensorflow packages can be checked here:
https://launchpad.net/~gichan-jang/+archive/ubuntu/ppa-build-test

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -12,9 +12,9 @@ Build-Depends:
  python-numpy,
  python-wheel,
  python-setuptools,
- curl,
+ curl, libflatbuffers-dev, flatbuffers-compiler
```
Comment (kparichay) [inline debian/control:15]: tensorflow-lite-dev depends should also include `libflatbuffers-dev`

Comment (kparichay) [reply debian/control:15]: Also, this needs to be updated for tizen packaging?

Comment (gichan-jang) [reply debian/control:15]: tizen packaging (https://review.tizen.org/gerrit/#/admin/projects/platform/upstream/tensorflow) have dependency to flatbuffers-dev.
One problem is that Tizen and Debian packaging are managed in different repos.
Tizen provides tf-lite only and debian provides both tf and tf-lite.
What is a good policy for managing repo for tensorflow?

Comment (kparichay) [reply debian/control:15]: I think all changes and maintaining should be performed in `review.tizen.org`.
And this repo should just sync with review.tizen.org

Comment (kparichay) [reply debian/control:15]: Tizen packaging only provides tflite.
Doesnt meant that tizen repo cannot maintain debian packaging.

**Comment by myungjoo** [comment]
DO any updates at review.tizen.org and use this repo as a mirror.

**Pull request closed without merging**